airflow/providers/postgres/hooks/postgres.py

```diff
@@ -157,7 +165,7 @@ def _serialize_cell(cell, conn):
         """
         return cell
 
-    def get_iam_token(self, conn):
+    def get_iam_token(self, conn: connection) -> Tuple[str, str, int]:
```

Comment:
the arg `conn`  of  method `get_iam_token(...)`, is airflow connection or the direct connection to PostgreSQL ? 

1. like this `self.connection` is an Airflow Connection class so it provides information on what parameters should be used for establishing PostgreSQL connection.
2. like this  `self.conn` is direct connection to PostgreSQL.